### ISSUE TITLE
document whoosh search runner

### ISSUE THREAD

**Issue by Mrten** [saltstack/salt#4308]
Nowhere in the docs or default master config is it mentioned how to enable the whoosh search runner. I understand why (alpha), but if noone finds or uses it it'll be dead code forever.

```
You need to enable the whoosh search backend in the master config:

search: whoosh

And restart the master. Then you can send in any search term to search query.
```

**Comment by Mrten** [comment]
don't think whoosh is still a part of salt?

**Comment by thatch45** [comment]
Fixed by #41245

**Comment by retr0h** [comment]
> Fixed by #41245

🖖
